dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/jobs/steps/processors/SubSnpCoreFieldsToVariantProcessor.java

```diff
@@ -27,7 +29,12 @@ public IVariant process(SubSnpCoreFields subSnpCoreFields) throws Exception {
         String referenceAllele = subSnpCoreFields.getReferenceInForwardStrand();
         String alternateAllele = subSnpCoreFields.getAlternateInForwardStrand();
 
-        return null;
+        Region chromosomeRegion = subSnpCoreFields.getVariantCoordinates();
+
+        IVariant variant = new Variant(chromosomeRegion.getChromosome(), chromosomeRegion.getStart(),
+                                       chromosomeRegion.getEnd(), referenceAllele, alternateAllele);
+        return variant;
```

Comment:
Either don't build the `Variant` object and still return null, or add a comment to flag very clearly that the mapping is not complete yet because, for instance, the code doesn't take into account multi-allelic SubSNPs.